Mage/src/main/java/mage/players/Player.java

```diff
@@ -834,7 +839,27 @@ default List<Integer> getMultiAmount(Outcome outcome, List<String> messages, int
 
     boolean addCounters(Counter counter, UUID playerAddingCounters, Ability source, Game game);
 
-    void removeCounters(String name, int amount, Ability source, Game game);
+    void removeCounters(String counterName, int amount, Ability source, Game game);
+
+    /**
+     * Remove (Lose) all counters.
```

Comment:
So for Players, the action verb is `lose`. Should we keep remove or rename the methods?

Reply:
I prefer `lose` in that case